### DISCUSSION TITLE
Help wanted to understand PNG issue

### DISCUSSION THREAD

**Discussion by CubeSolutionsFrance** [ImageMagick/ImageMagick#6804] [Help]
### ImageMagick version

v4.0.30319

### Operating system, version and so on

Windows 10

### Description

![848541](https://github.com/ImageMagick/ImageMagick/assets/74758945/7df91184-0d46-4aff-9e0f-833b2dff1cfd)
Hi,
I have a problem with several PNG images like the one attached to that post.
When displaying this image, it seems to be a full transparent image.
Nevertheless, the image is a real image of a bier bottle.
When converting it to JPG, this is a real bier bottle image.
It seems that my PNG file is "corrupted" or something like that.
But I don't know how to figure it out and how to work with it without having to convert it to JPG.
Have you ever experienced such behavior ?
I'm not the creator of this image so I would like to know what is wrong with it.
Any tip is fully welcome.
Thanks a lot in advance.
Kind regards,

**Comment by CubeSolutionsFrance** [comment]
[848541.zip](https://github.com/ImageMagick/ImageMagick/files/13039769/848541.zip)

**Comment by snibgo** [comment]
The file `848541.png` is not corrupted. It is entirely transparent. We can see this from the text output of `magick 848541.png -verbose info:`. However, each transparent pixel has a sensible colour. So we can remove transparency like this:
```
magick 848541.png -alpha off out.png
```

**Reply by CubeSolutionsFrance** [reply]
@snibgo  thanks a lot for your analysis and your answer. It helps a lot.
I have used the verbose option. But which info can let me understand that it is entirely transparent ?
Thanks again

**Comment by snibgo** [comment]
```
    Alpha:
      min: 0  (0)
      max: 0 (0)
      mean: 0 (0)
      median: 0 (0)
      standard deviation: 0 (0)
      kurtosis: -3
      skewness: 0
      entropy: 0
```
The minimum and maximum values of alpha are 0, so it is entirely zero, so all pixels are fully transparent.

We can also do this:
```
magick 848541.png -format "minA=%[fx:minima.a]\nmaxA=%[fx:maxima.a]" info:
minA=0
maxA=0
```

**Reply by CubeSolutionsFrance** [reply]
Crystal clear. Thanks a lot.
I think I will use this as a test to sort my different images.
